plugins/modules/net_tools/nmcli.py

```diff
@@ -73,7 +74,27 @@
         description:
             - The IPv4 gateway for this interface.
             - Use the format C(192.0.2.1).
+            - This parameter is mutually_exclusive with never_default4 parameter.
         type: str
+    routes4:
+        description:
+            - The list of ipv4 routes.
+            - Use the format '192.0.3.0/24 192.0.2.1'
+        type: list
+        elements: str
+        version_added: 1.3.0
```

Comment:
1.3.0 was released a few days ago.

Reply:
Bad luck for me. Ok then I will also update that 🙂